app/controllers/api/calls_controller.rb

```diff
@@ -16,7 +17,8 @@ def create
 
   def call_params
     params.require(:call)
-      .permit(:member_phone_number, :target_id)
+      .permit(:member_phone_number, :target_id, :target_title, :target_name,
+              :target_phone, :target_country_code, :target_phone_extension, :checksum)
```

Comment:
I was under the assumption that the we'd receive an encrypted token / checksum that, once base64 decoded and then decrypted, would contain the target object?